internal/connectors/plugins/public/powens/webhooks.go

```diff
@@ -261,6 +261,21 @@ func (p *Plugin) handleConnectionSynced(ctx context.Context, req models.Translat
 			at = webhook.Connection.LastUpdate
 		}
 
+		amountString := strconv.FormatFloat(account.Balance, 'f', -1, 64)
```

Comment:
I know you aren't the one who introduced the decimal handling for Powens, as the transactions webhook extraction is doing similar, but I am a bit afraid of floating point precision errors here.

Can we double check that powens uses `json.Number` in the unmarshal step and then treats those numbers as strings until we can convert the amounts to their value in minors (Eg. $20.07 -> 2007). Mangopay, Moneycorp, Wise and Qonto for example are doing this.